fund/qubes-donations-eth-address.txt

```diff
@@ -1,12 +1,16 @@
-The Qubes OS Project has one official Ethereum (ETH) donation address:
+The Qubes OS Project has one official Ethereum Mainnet donation address:
```

Comment:
FYI, the reason this doesn't follow the pattern of the other two (`bitcoin (BTC)` and `monero (XMR)`) is because here we're saying that this is a donation address for the Ethereum Mainnet _network_. We're not saying that it's a donation address for a specific type of cryptocurrency (`ether (ETH)`), because this address is actually capable of accept donations in many other types of cryptocurrency as well (e.g., USDT, USDC, and DAI), as long as they fully conform to ERC-20 and are on Ethereum Mainnet.